### PR TITLE
Breakpoint clear

### PR DESCRIPTION
**Pull request by stuartromanek**
## Summary

- Adds a Clear Breakpoint button when breakpoint prev is activated
- Moves Breakpoint controls to the left side of the context bar

## What are the specific steps to test this change?

https://github.com/user-attachments/assets/a79fcb17-bf7f-4adb-9aff-f163a9166816

1. Activate a breakpoint
2. See the X button appear
3. click it and be returned to full preview

**⚠️ NOTE**
The intended responsive effect of the breakpoint preview UI is that the shortcuts container will disappear at small screen sizes. But, due to an issue with the way the breakpoint preview webpack media query loader currently works, ALL media queries (including apos admin styles) get scoped to the breakpoint preview mode and therefore the shortcuts container disappears _whenever_ breakpoint preview is activated. This is _not_ the intended effect, but as other admin elements are already affected by this, this PR does not try to address it.

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [x] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [ ] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by boutell** [comment]
ooo nice

File: modules/@apostrophecms/ui/ui/apos/scss/mixins/_theme_mixins.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $brand-green: #00bf9a;
 
 $spacing-base: 10px;
 $spacing-half: math.div($spacing-base, 2);
+$spacing-one-quarter: math.div($spacing-base, 4);
 $spacing-three-quarters: math.div($spacing-base, 4) * 3;
```
Comment (ValJed) [inline modules/@apostrophecms/ui/ui/apos/scss/mixins/_theme_mixins.scss:12]: [Detail]
Could reuse `$spacing-one-quarter` in this line.

**Review by ValJed — CHANGES_REQUESTED** [review]
Looks good, just a comment to remove.
Might worth running cypress tests before to merge.

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue [hunk 2]
```diff
@@ -50,13 +50,13 @@
...
       />
-      <TheAposContextBreakpointPreviewMode
+      <!-- <TheAposContextBreakpointPreviewMode
         v-if="isBreakpointPreviewModeEnabled"
         :screens="breakpointPreviewModeScreens"
         :resizable="breakpointPreviewModeResizable"
         @switch-breakpoint-preview-mode="addContextLabel"
         @reset-breakpoint-preview-mode="removeContextLabel"
-      />
+      /> -->
```
[the remark this answers is not in the document]
Comment (stuartromanek) [reply modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue]: removed

**Comment by stuartromanek** [comment]
Tests running here https://github.com/apostrophecms/testbed/actions/runs/11388565860

**Comment by stuartromanek** [comment]
> Tests running here https://github.com/apostrophecms/testbed/actions/runs/11388565860

@ValJed tests for `import-export` are failing but they are failing the same way they are in `[main](https://github.com/apostrophecms/testbed/actions/runs/11375941148/job/31647661948)`

`¯\_(ツ)_/¯` ?

**Pull request merged**